test/test_helper.rb

```diff
@@ -8,6 +8,15 @@
 require 'mocha/mini_test'
 
 module CategoryItemsSetup
+  def create_locations
+    locations = [Location.create(name: 'Capitol Hill'),
```

Comment:
Useless assignment to variable - `locations`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.